lib/stages/rfiBroadcast.cpp

```diff
@@ -46,7 +46,12 @@ REGISTER_KOTEKAN_STAGE(rfiBroadcast);
 
 rfiBroadcast::rfiBroadcast(Config& config, const std::string& unique_name,
                            bufferContainer& buffer_container) :
-    Stage(config, unique_name, buffer_container, std::bind(&rfiBroadcast::main_thread, this)) {
+    Stage(config, unique_name, buffer_container, std::bind(&rfiBroadcast::main_thread, this)),
+    sample_counter(Metrics::instance().add_counter("kotekan_rfibroadcast_sample_total", unique_name,
+                                                   {"freq_id"})),
+    flagged_sample_counter(Metrics::instance().add_counter(
+        "kotekan_rfibroadcast_flagged_sample_total", unique_name, {"freq_id"})) {
```

Comment:
To align this with `RfiFrameDrop` (i.e. the second stage) you might call this `kotekan_rfibroadcast_dropped_sample_total`.